### PR TITLE
Put contact mode in edit view by performing a selector on it delayed

### PR DESCRIPTION
**Pull request by rhoglund**
Open native contact form in edit mode

### CODE REVIEW

File: ios/RCTContacts/RCTContacts.m [hunk 1]
```diff
@@ -386,14 +405,14 @@ -(NSString *) getFilePathForThumbnailImage:(NSString *)recordID
 
     [self updateRecord:contact withData:contactData];
 
-    CNContactViewController *controller = [CNContactViewController viewControllerForNewContact:contact];
-
-    controller.delegate = self;
+    CNContactViewController *contactViewController = [CNContactViewController viewControllerForNewContact:contact];
```
Comment (cmasus) [inline ios/RCTContacts/RCTContacts.m]: Maybe just revert these lines to keep the PR shorter. No actual changes have been made except variable name changes.

Comment (rhoglund) [reply ios/RCTContacts/RCTContacts.m]: sure

Comment (rhoglund) [inline ios/RCTContacts/RCTContacts.m]:
```suggestion
    CNContactViewController *contactViewController = [CNContactViewController viewControllerForNewContact:contact];
```

File: ios/RCTContacts/RCTContacts.m [hunk 2]
```diff
@@ -404,52 +423,92 @@ -(NSString *) getFilePathForThumbnailImage:(NSString *)recordID
...
+        contactViewController.delegate = self;
+
+
         dispatch_async(dispatch_get_main_queue(), ^{
-            // TODO: Change so this is not opened as a modal. We want the back button!
-            UINavigationController* navigation = [[UINavigationController alloc] initWithRootViewController:controller];
-            UIViewController *viewController = (UIViewController*)[[[[UIApplication sharedApplication] delegate] window] rootViewController];
-            [viewController presentViewController:navigation animated:YES completion:nil];
-            
+            UINavigationController* navigation = [[UINavigationController alloc] initWithRootViewController:contactViewController];
```
Comment (cmasus) [inline ios/RCTContacts/RCTContacts.m:449]: I have tested it and I can see one small thing that we probably can ignore. If I press cancel the Done button in the top right corner flicks to Edit before the view disappears.

Comment (rhoglund) [reply ios/RCTContacts/RCTContacts.m:449]: Ok, yes I think we will ignore it.

**Pull request merged**